Gemfile

```diff
@@ -17,6 +17,7 @@ group :test do
     gem "chef-zero", "~> 15"
     gem "chef", "~> 15"
   end
+  gem "knife"
```

Comment:
@nikhil2611 Why this `knife` gem is required and if it is needed why no version is mentioned